front/app/components/admin/FormResults/FormResultsQuestion/SentimentQuestion/components/Comments.tsx

```diff
@@ -77,7 +77,6 @@ const Comments = ({
                 <Analysis
                   customFieldId={customFieldId}
                   textResponsesCount={textResponses.length}
-                  hasOtherResponses={true}
```

Comment:
Not quite sure why this has been removed. Sure there's a good reason but can't figure it out!

Reply:
It shouldn't have been there in the first place actually :sweat_smile: Was a result of me re-using/copying some of the code from the "Other" response for select questions originally.